sky/engine/core/script/dart_init.cc

```diff
@@ -420,7 +430,7 @@ void InitDartVM() {
   }
   CHECK(Dart_SetVMFlags(args.size(), args.data()));
 
-  {
+  if (IsDevelopMode()) {
```

Comment:
It likely does because the IsDevelopMode is defined in this compilation unit and the compiler can determine at compile time this whole branch is or is not taken.